src/types/drupal/field_type.d.ts

```diff
@@ -31,8 +31,7 @@ export interface FieldLink {
   uri: string //e.g. `entity:node/2424`
   url?: string //e.g. `/outreach-and-events/events/2424`
   title: string
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  options: any[]
+  options: unknown[]
```

Comment:
Not actually related to my changes, but I found this while spelunking and decided it should be `unknown` instead of `any`. 🤷